dotnet/src/webdriver/Firefox/FirefoxExtension.cs

```diff
@@ -48,6 +48,7 @@ public class FirefoxExtension
         /// <remarks>WebDriver attempts to resolve the <paramref name="fileName"/> parameter
         /// by looking first for the specified file in the directory of the calling assembly,
         /// then using the full path to the file, if a full path is provided.</remarks>
+        /// <exception cref="ArgumentNullException">If <paramref name="fileName"/> is <see langword="null"/>.</exception>
```

Comment:
This is already unreachable with `null` using normal means, but this constructor is public-facing so we need the validation.